### DISCUSSION TITLE
Doesn't generate js (SharedWorker -  CRA - webPack4 -  signalR)

### DISCUSSION THREAD

**Discussion by SmashPlusUltra** [webpack/webpack#15777] [Q&A]
Hi! I'm using all these things **SharedWorker -  CRA - webPack4 -  signalR and react-app-rewired**, when I do 
npm run build -> doesn't generate any file.
if I comment the following
`const connection = new signalR.HubConnectionBuilder().withUrl('test').withAutomaticReconnect().build();`
It works.

I'll attach the source code, you can dowload and do
npm install
npm run build 
And you'll see
![image](https://user-images.githubusercontent.com/53838441/167038895-f17374e2-6fb1-4b0a-8d71-dce04d8f1368.png)
[myApp.zip](https://github.com/webpack/webpack/files/8636470/myApp.zip)

If I do npm start, the app work without problem and use the library without any problem.  

Any help would be very appreciate

**Comment by vankop** [comment]
please ask in CRA, it does not seems related to webpack.. ( no warnings from webpack, it looks like only eslint warnings )

**Reply by SmashPlusUltra** [reply]
Ok! I'll changed it and upload a new [discussion](https://github.com/webpack/webpack/discussions/15791) using just webpack & babel so no CRA is in the project. Thanks!
